tools/ubergen/bin/ubergen.ts

```diff
@@ -402,6 +408,37 @@ async function copyOrTransformFiles(from: string, to: string, libraries: readonl
   await Promise.all(promises);
 }
 
+/**
+ * Rewrites the imports in README.md from v1 ('@aws-cdk/...') to v2 ('aws-cdk-lib').
+ * Uses the module imports (import { aws_foo as foo } from 'aws-cdk-lib') for module imports,
+ * and "barrel" imports for types (import { Bucket } from 'aws-cdk-lib/aws-s3').
+ */
+async function rewriteReadmeImports(fromFile: string): Promise<string> {
+  const readmeOriginal = await fs.readFile(fromFile, { encoding: 'utf8' });
+  return readmeOriginal
+    // import * as s3 from '@aws-cdk/aws-s3'
+    .replace(/^(\s*)import \* as (.*) from '@aws-cdk\/(.*)';(\s*)$/gm, rewriteCdkImports)
```

Comment:
do we also need to take care of double quotes, besides single quotes?

Reply:
Good call :+1:. There are 14 imports across the entire codebase that use double-quotes. I'm tempted to fix them and leave the code as-is, but I suppose that's more fragile than just handling the quotes properly here.